src/video.tsx

```diff
@@ -17,12 +17,22 @@ export const RATIO = {
 
 const useStyles = createUseStyles(() => ({
     video: {
-        borderRadius: ({noBorderRadius}) => (noBorderRadius ? 0 : 4),
         display: 'block',
+        position: 'absolute',
+        width: '100%',
+        height: '100%',
+        top: 0,
+        left: 0,
+        objectFit: 'cover',
+    },
+
+    wrapper: {
+        borderRadius: ({noBorderRadius}) => (noBorderRadius ? 0 : 4),
+        overflow: 'hidden',
         maxWidth: '100%',
         maxHeight: '100%',
-        objectFit: 'cover',
-        aspectRatio: ({aspectRatio}) => aspectRatio ?? 'unset',
+        position: 'relative',
+        paddingTop: ({aspectRatio}) => (aspectRatio ? `${100 / aspectRatio}%` : 'initial'),
```

Comment:
I'd implement the same fix for images

Reply:
Yes, I'm on it. But I don't like the implementation. I think I'd try to feature-detect the aspect-ratio support and only apply this hack when needed. I don't wan't every Image to have a div wrapper :(